src/blob/index/simple.rs

```diff
@@ -128,7 +123,7 @@ impl Simple {
         ))
     }
 
-    pub async fn load_records(mut file: &File) -> Result<Vec<RecordHeader>> {
+    pub async fn load_records(file: &File) -> Result<Vec<RecordHeader>> {
         trace!("seek to file start");
         file.seek(SeekFrom::Start(0)).await?;
```

Comment:
Is there a race condition with line 136? As after you seek to the position someone can change location before you actually read it. Or it is done during init state?

Reply:
same question to each place with `seek` operations

Reply:
seek operations removed in #49